cli/cmd/package_manifest.go

```diff
@@ -258,17 +259,62 @@ func (c *cliState) GetOSInfo() (*OS, error) {
 		"arch", runtime.GOARCH,
 	)
 
-	f, err := os.Open(osReleaseFile)
-	if err != nil {
-		msg := `unsupported platform
+	_, err := os.Stat(osReleaseFile)
+	if err == nil {
```

Comment:
we should use this function to check if the file exist
```suggestion
	if fileExists(osReleaseFile) {
```